src/modules/UI/scenes/TransactionList/style.js

```diff
@@ -65,15 +65,33 @@ export const styles = {
   },
 
   // end of searchbar stuff
-
+  touchablBalanceBox: {
+    height: 200
+  },
   currentBalanceBox: {
     flex: 1,
     justifyContent: 'center'
   },
+  totalBalanceWrap: {
```

Comment:
what does this do?

Reply:
It is a wrap around certain elements in the TxList Balance Box.

Reply:
It styles a portion of the BalanceBox. I modified the structure of the JSX and that means adjustments to the styling